site/.vitepress/genFeed.ts

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+import { writeFileSync } from 'fs';
+import { Feed } from 'feed';
+import { createContentLoader, type SiteConfig } from 'vitepress';
+
+const baseUrl = `https://wrensecurity.org`;
+
+export async function genFeed(config: SiteConfig) {
+  const feed = new Feed({
+    title: 'Wren Security',
+    description: 'The official blog for Wren Security projects',
+    id: baseUrl,
+    link: baseUrl,
+    language: 'en',
+    image: 'https://wrensecurity.org/wrensec-logo.png',
+    favicon: `${baseUrl}/favicon.ico`,
+    copyright:
+      'Copyright © 2017-present Wren Security'
+  });
+
+  const posts = await createContentLoader('blog/**/*.md', {
+    excerpt: true,
+    render: true
+  }).load();
+
+  // Filter everything that's not of type `article` (e.g. index.md) // Vyfiltrovat o index v rámci url
```

Comment:
Please remove your internal notes and improve the comment itself.

Reply:
I removed and rewrote the comment.